### ISSUE TITLE
Activision Studios

### ISSUE THREAD

**Issue by GreatdayJay** [gta-chaos-mod/ChaosModV#1804]
Charges the player money for everything and If they don't have sufficient funds it does not allow it

**Comment by RosyMiranto18** [comment]
Isn't that more of the EA Meme? You should rename the effect to "GTA by EA"

**Comment by GreatdayJay** [comment]
> Isn't that more of the EA Meme? You should rename the effect to "GTA by EA"

Agreed, I was just thinking warzone but more people know about EA's scams
